main.py

```
@@ -6160,145 +6160,147 @@ def __init__(self):
-                "red": "🔴",
-                "black": "⚫",
-                "green": "🟢",
-            }
+            random_times = [0.1] * random.randint(2, 3)
+            base_times = [0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1, 1.1, 1.2, 1.5]
+            wait_times = random_times + base_times
+
+            padding = 4
+            roulette_spin_times = len(wait_times) + padding
+            
+            start_point = random.randint(0, 37)
```

Ruff (0.15.5)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)